src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/RotatingStar.cpp

```diff
@@ -614,9 +691,10 @@ RotatingStar::variables(
   using std::max;
   return {hydro::relativistic_specific_enthalpy(
       Scalar<DataType>{DataType{max(1.0e-300, get(rest_mass_density))}},
-      equation_of_state_.specific_internal_energy_from_density(
-          rest_mass_density),
-      equation_of_state_.pressure_from_density(rest_mass_density))};
+      equation_of_state_->specific_internal_energy_from_density(
```

Comment:
Same as the pressure, call the other function so you don't have multiple copies of the magic number floating around.